test/server/cards/02_SHD/leaders/FennecShandHonoringTheDeal.spec.ts

```diff
@@ -0,0 +1,103 @@
+describe('Fennec Shand, Honoring the Deal', function () {
+    integration(function (contextRef) {
+        describe('Fennec Shand\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        base: 'echo-base',
+                        leader: 'fennec-shand#honoring-the-deal',
+                        hand: ['reinforcement-walker', 'rebel-pathfinder', 'alliance-xwing', 'modded-cohort'],
+                        resources: 4
+                    },
+                    player2: {
+                        groundArena: ['isb-agent'],
+                    }
+                });
+            });
+
+            it('should allow the player to play a unit with cost 4 or less, paying its cost, and give it ambush', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.fennecShand);
+                expect(context.player1).toBeAbleToSelectExactly([context.rebelPathfinder, context.allianceXwing, context.moddedCohort]);
```

Comment:
Can you change modded cohort to be a red 4-cost card so we can confirm that it's looking at printed cost without aspect penalties